src/index.js

```diff
@@ -234,6 +239,12 @@ export default function depcheck(rootDir, options, callback) {
       dependencies: minus(deps, result.used),
       devDependencies: minus(devDeps, result.used),
       missing: minus(result.used, allDeps).filter(dep => builtInModules.indexOf(dep) === -1),
+      used: Object.keys(result.hits).reduce((obj, current) => {
+        return result.hits[current].reduce((o, d) => {
```

Comment:
I think ESLint should fail your `o` and `d` variable names. I don't know why CI still pass. :(